packages/@aws-cdk/region-info/build-tools/fact-tables.ts

```diff
@@ -134,8 +134,12 @@ export const DLC_REPOSITORY_ACCOUNTS: { [region: string]: string } = {
   'cn-northwest-1': '727897471807',
 };
 
-// https://docs.aws.amazon.com/app-mesh/latest/userguide/envoy.html
-export const APPMESH_ECR_ACCOUNTS: { [region: string]: string } = {
+/**
+ * ECR Repositories where App Mesh Images are hosted
+ *
+ * @see https://docs.aws.amazon.com/app-mesh/latest/userguide/envoy.html
+ */
+export const APPMESH_REPOSITORY_ACCOUNTS: { [region: string]: string} = {
```

Comment:
Any reason we're changing the name from `APPMESH_ECR_ACCOUNTS` to `APPMESH_REPOSITORY_ACCOUNTS`? Maybe leaving it as-is is a better idea?

Reply:
Yes, I made this PR before those existed. I will remove